docs/src/tutorials/pluscal-tutorial.md

```diff
@@ -0,0 +1,241 @@
+# Tutorial on checking Pluscal specifications with Apalache
```

Comment:
I think the official spelling is Plus**C**al. I'm only marking the first occurrence.